hermes/src/main/java/in/raveesh/hermes/Util.java

```diff
@@ -0,0 +1,16 @@
+package in.raveesh.hermes;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+/**
+ * Created by sahil on 15/5/15.
+ */
+public class Util {
+    public static final String LOG_TAG = "in.raveesh.hermesexample";
+
+    @SuppressLint("LongLogTag")
+    public static void log(String message) {
```

Comment:
I'm a n00b when it comes to annotations. Can you explain the benefits of this vs a regular Log, other than typing down a tag each and every time?